src/nvim/po/de.po

```diff
@@ -153,476 +396,1466 @@ msgstr ""
 
-#: ../ex_cmds.c:1189
-msgid "E135: *Filter* Autocommands must not change current buffer"
-msgstr "E135: *Filter*-Autokommandos d�rfen den aktuellen Puffer nicht �ndern"
+#: ../eval.c:159
+msgid "E713: Cannot use empty key for Dictionary"
+msgstr "E713: Schl�ssel f�r Dictionary darf nicht leer sein"
 
-#: ../ex_cmds.c:1239
-msgid "[No write since last change]\n"
-msgstr "[Kein Schreiben seit der letzten �nderung]\n"
+#: ../eval.c:160
+msgid "E714: List required"
+msgstr "E714: List ben�tigt"
 
-#: ../ex_cmds.c:1418
-#, c-format
-msgid "%sviminfo: %s in line: "
-msgstr "%sviminfo: %s in Zeile: "
+#: ../eval.c:161
+msgid "E715: Dictionary required"
+msgstr "E715: Dictionary ben�tigt"
 
-#: ../ex_cmds.c:1425
-msgid "E136: viminfo: Too many errors, skipping rest of file"
-msgstr "E136: Viminfo: Zu viele Fehler; �berspringe Rest der Datei"
+#: ../eval.c:162
+#, fuzzy
+msgid "E928: String required"
+msgstr "E397: Dateiname wird ben�tigt"
 
-#: ../ex_cmds.c:1452
+#: ../eval.c:163
 #, c-format
-msgid "Reading viminfo file \"%s\"%s%s%s"
-msgstr "Lesen der viminfo-Datei \"%s\"%s%s%s"
+msgid "E118: Too many arguments for function: %s"
+msgstr "E118: Zu viele Argumente f�r Funktion: %s"
 
-#: ../ex_cmds.c:1454
-msgid " info"
-msgstr " Information"
+#: ../eval.c:164
+#, c-format
+msgid "E716: Key not present in Dictionary: %s"
+msgstr "E716: Schl�ssel nicht in Dictionary enthalten: %s"
 
-#: ../ex_cmds.c:1455
-msgid " marks"
-msgstr " Marken"
+#: ../eval.c:166
+#, c-format
+msgid "E122: Function %s already exists, add ! to replace it"
+msgstr "E122: Funktion %s existiert bereits; zum Ersetzen ! hinzuf�gen"
 
-#: ../ex_cmds.c:1456
-msgid " oldfiles"
-msgstr " oldfiles"
+#: ../eval.c:167
+msgid "E717: Dictionary entry already exists"
+msgstr "E717: Dictionary-Eintrag existiert bereits"
 
-#: ../ex_cmds.c:1457
-msgid " FAILED"
-msgstr " FEHLGESCHLAGEN"
+#: ../eval.c:168
+msgid "E718: Funcref required"
+msgstr "E718: Funktionsreferenz ben�tigt"
+
+#: ../eval.c:169
+msgid "E719: Cannot use [:] with a Dictionary"
+msgstr "E719: Kann [:] nicht mit Dictionary verwenden"
 
-#. avoid a wait_return for this message, it's annoying
-#: ../ex_cmds.c:1535
+#: ../eval.c:170
 #, c-format
-msgid "E137: Viminfo file is not writable: %s"
-msgstr "E137: Viminfo-Datei ist nicht schreibbar: %s"
+msgid "E734: Wrong variable type for %s="
+msgstr "E734: Falscher Typ der Variable f�r %s="
 
-#: ../ex_cmds.c:1620
+#: ../eval.c:171
 #, c-format
-msgid "E138: Can't write viminfo file %s!"
-msgstr "E138: Schreiben der viminfo-Datei %s ist nicht m�glich!"
+msgid "E130: Unknown function: %s"
+msgstr "E130: Unbekannte Funktion: %s"
 
-#: ../ex_cmds.c:1629
+#: ../eval.c:172
 #, c-format
-msgid "Writing viminfo file \"%s\""
-msgstr "Schreibe viminfo-Datei \"%s\""
+msgid "E461: Illegal variable name: %s"
+msgstr "E461: Unzul�ssiger Name der Variable: %s"
+
+#: ../eval.c:173
+msgid "E806: using Float as a String"
+msgstr "E806: Float als String verwendet"
```

Comment:
'integer' was translated to 'Ganzzahl', why not translationg 'float' to 'Gleitkommazahl', too?